src/main/java/org/spongepowered/api/text/sink/MessageSinks.java

```diff
@@ -24,16 +24,29 @@
  */
 package org.spongepowered.api.text.sink;
 
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import org.spongepowered.api.Sponge;
 import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.text.Text;
 
+import java.util.Collection;
+import java.util.Optional;
 import java.util.Set;
+import java.util.function.BiFunction;
 
 /**
  * Useful building blocks for message sinks.
  */
 public class MessageSinks {
 
-    private static final MessageSinkFactory factory = null;
+    public static final MessageSink TO_NONE = ImmutableSet::of;
+    public static final MessageSink TO_ALL = () -> {
+        Set<CommandSource> recipients = Sets.newHashSet(Sponge.getGame().getServer().getOnlinePlayers());
+        recipients.add(Sponge.getGame().getServer().getConsole());
+        return ImmutableSet.copyOf(recipients);
```

Comment:
If you return a `ImmutableSet` here anyway then you could just use the `ImmutableSet.Builder` here.

````java
final ImmutableSet.Builder<CommandSource> builder = ImmutableSet.builder();
        final Server server = Sponge.getGame().getServer();
        builder.addAll(server.getOnlinePlayers());
        builder.add(server.getConsole());
        return builder.build();
````
or
````java
final Server server = Sponge.getGame().getServer();
return ImmutableSet.builder()
        .addAll(server.getOnlinePlayers())
        .add(server.getConsole())
        .build();
````